spec/features/article_searching_spec.rb

```diff
@@ -67,4 +68,40 @@
     expect(page).to have_css('h1', text: 'The title of the document')
     expect(current_url).to match(%r{/article/abc123})
   end
+
+  describe 'breadcrumbs', js:true do
+   scenario 'start over button returns users to articles home page' do
+     stub_article_service(docs: [SolrDocument.new(id: 'abc123')])
+     visit article_index_path
+
+     within '.search-form' do
+       fill_in 'q', with: 'kittens'
+       find('#search').trigger('click')
+     end
+
+     expect(page).to have_css('.appliedFilter', text: /kittens/)
+
+     within '#appliedParams' do
+       find('a.btn', text: /Start over/).trigger('click')
+       expect(page).to have_current_path(article_index_path)
+       expect(page).not_to have_css('.appliedFilter', text: /kittens/)
```

Comment:
You may want to do this outside of the `within` (or even in a separate `within` block below it).   Clicking on `Start over` has changed the page so I have found that the tests can complain about the element going way when that happens.